manifests/cf-manifest/manifest/010-cf-jobs.yml

```diff
@@ -222,6 +224,9 @@ jobs:
       - name: cf
     update:
       serial: true
+    properties:
+      bbs:
+        url: (( grab meta.bbs.api_location  ))
```

Comment:
So we have two checks running on database VMs both submitting `cf.bbs.Health` and `cf.bbs.ResonseTime`. That's fine as check is using `bbs.service.cf.internal:8889` to connect.